speaker/main.go

```diff
@@ -15,10 +15,15 @@
 package main
 
 import (
+	"crypto/sha256"
 	"flag"
 	"fmt"
+	golog "log"
```

Comment:
Is there a reason we need to use the std lib log instead of `gokitlog`

Reply:
For the logs, MemberList accept either an io.Writer or a log.Logger. I prefer the log.Logger so can avoid having the date 2 times, and add the filename:line.
In the end we pass a golog.Logger, with a special io.Writer that use gokitlog (please tell me if we can had some more levels :P)